### DISCUSSION TITLE
Can I run GraalVM native-image app on ARMv7 (32bit) device?

### DISCUSSION THREAD

**Discussion by mkarczewski85** [oracle/graal#3201] [Q&A] [answered]
Is it possible to run GraalVM native-image of Quarkus framework app on ARMv7 (32bit) device? I know that GraalVM doesn't support 32bit environment but I wonder if there is some way to run binary with compiled GraalVM native-image on such. Is there some way to configure LLVM, used by GraalVM to achieve that? IoT device we preparing has to have some web application (accessed locally) with simple dashboard. Since device storage is too small to install any Java JDK >= 8, I wonder if GraalVM native-image would do the work.

**Comment by thomaswue** [comment, marked as answer]
We do not support this platform at the moment, but one of our partners MicroDoc is working on this (https://www.microdoc.com/graal-vm/).

**Reply by mkarczewski85** [reply]
Great news. Thanks
